### PR TITLE
dashboard with cards and searchbar

### PR DESCRIPTION
**Pull request by juanmeyer94**
## Info / Context

- The dashboard has been created, including the cards component and the navbar with search bar.

## Changes

-  Remove the standard HTML components of the Vite app and add the new components.
- Add the search bar and the cards to the "components" folder.

## How do you feel about this PR? (GIF)

giphy

### CODE REVIEW

File: apps/client/src/components/cloudinary/UploadWidget.jsx [hunk 1]
```diff
@@ -1,32 +1,24 @@
...
+        if (!error && result && result.event === "success") {    
           // document
           // .getElementById("uploadedimage").src = result.info.secure_url
           //   .setAttribute("src", result.info.url); NO BORRAR POR EL MOMENTO
           this.props.handleImageUpload(result.info.url);
-          //this.props.handleImageId(result.info.public_id);
-
-            
-          
+          //this.props.handleImageId(result.info.public_id);
```
Comment (nujovich) [inline apps/client/src/components/cloudinary/UploadWidget.jsx]: Borrar comentario!

**Review by nujovich — CHANGES_REQUESTED** [review]

File: apps/client/package.json [hunk 2]
```diff
@@ -12,20 +12,21 @@
   "dependencies": {
     "@cloudinary/react": "^1.11.2",
     "@cloudinary/url-gen": "^1.10.1",
-    "@uploadthing/solid": "^5.1.2",
-    "cloudinary-core": "^2.13.0",
-    "solidjs-dropzone": "^1.0.0",
-    "uploadthing": "^5.3.3",
     "@radix-ui/react-alert-dialog": "^1.0.4",
     "@radix-ui/react-slot": "^1.0.2",
+    "@uploadthing/solid": "^5.1.2",
```
Comment (cesmunoz) [inline apps/client/package.json]: Si no vamos a usar uploadthing creo que podemos borrarlo

File: apps/client/src/components/serachbar/SearchBar.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    const [searchValue, setSearchValue] = useState([])
+
+    const handleInputOnChange = (event) => {
+        event.preventDefault()
+        setSearchValue(event)
+    }
+    
+    const onSubmit = () => {
+
+    }
```
Comment (cesmunoz) [inline apps/client/src/components/serachbar/SearchBar.jsx:16]: Se puede borrar esta funcion?

**Review by cesmunoz — APPROVED** [review]
Deje unos pequeños comentarios pero lo veo bien!
Buen trabajo!

File: apps/client/src/components/cloudinary/UploadWidget.jsx (hunk 1 above)
Comment (juanmeyer94) [reply apps/client/src/components/cloudinary/UploadWidget.jsx]: Arreglado!! 💪

**Pull request merged**